src/io.jl

```diff
@@ -70,10 +72,15 @@ Load Unfold models existing in a derivatives_subfolder in your BIDS root folder.
 
 # Keywords
 derivatives_subfolder (String::"Unfold"): Defines in which subfolder of bids_root/derivatives to look for Unfold models.
+
 lazy (Bool::false): Do not actually load the dataset into memore if true, only return a dataframe with paths
+
 generate_Xs (Bool::true): Do not recreate the designmatrix; improves loading time.
```

Comment:
I think here, and with all other keywords, you should put them into ` backticks, so that e.g. generate_Xs the underscore is not recognized as a "beginning of italic"

This is a todo for all docustrings